### ISSUE TITLE
 Standard names: Residuum of soil water content

### ISSUE THREAD

**Issue by rpnaut** [cf-convention/discuss#109]
**Proposer's name** Ronny Petrik and Klaus Keuler and Burkhardt Rockel
**Date** 20.05.2021.
In climatological applications the budget of the soil water is of big importance. As simulations are done over many decades, the conservation of soil water mass (and also other mass quantities and the energy) is required for a robust and physicall-consistent simulation. Up to now, only the budgets of ice sheets are considered with the CF standard names, e.g. _land_ice_surface_specific_mass_balance_flux_ or _land_ice_surface_specific_mass_balance_rate_

**- Term** residuum_of_soil_water_content
**- Description** "Water" means water in all phases. "Content" indicates a quantity per unit area. The mass content of water in soil refers to the vertical integral from the surface down to those layer of the soil model which is the lowermost hydrological active layer. For the content between specified levels in the soil, standard names including "content_of_soil_layer" are used. "Residuum" is defined by the difference between the tendency of the soil moisture content over a specific time period (left hand side of soil water equation) and the sum of the sources and sinks of soil water over the same time period (right hand side of equation). A positive value means that the soil becomes falsely wetter in a model and a negative value means the soil becomes falsely dryer. It should be noted, that the residuum has to be calculated excluding climatological layers where the temperature or 
the soil water content is kept to a climatological value or a value given by another soil layer.
**- Units** kg m-2.

**Comment by GeyerB** [comment]
The new name is needed in the modelling community, thanks for the proposal!

It might be better to use `residuum_of_soil_water_mass_balance` instead of residuum_of_soil_water_content as the subject of study is the mass_balance.

My attempt to shorten the description was not successful, but perhaps the following is easier to understand:
-**Description** "Water" means water in all phases. The "mass" of water in soil refers to the vertical integral from the surface down to the soil bottom. In models, "soil bottom" refers to the layer of the soil model which is the lowest hydrologically active layer (explicitly excluding climatological layers with prescribed temperature or water content). "Residuum" is defined as the difference between the tendency of the soil moisture content over a specific time period and the sum of the sources and sinks of soil water over the same time period. Non-zero values indicate a non-closed water balance. Positive values mean that the soil in a model is falsely becoming wetter and negative values mean that the soil is falsely becoming dryer.

**Comment by JonathanGregory** [comment]
Dear all

Thanks for the proposal. I support the idea. I have a few comments.

  * Concerning the description, I would say that this might not only apply to models. You could have an observational budget, in principle.
  * Although the description is clear about it, the proposed standard name doesn't indicate the sign convention.
  * Without the description, it's possibly unclear what "residuum" means.

In view of the above, I would suggest something more explicit for the name, such as `difference_between_change_in_soil_moisture_content_and_sum_of_estimated_contributions`. In other standard names, `difference_between_X_and_Y` means X minus Y. By inserting "estimated" I mean to imply that there is room for error. There must be an error of some sort if there is a significant difference! If it's any clearer, we could say `net_change`, although that's what `change` alone means.

Best wishes

Jonathan

**Comment by GeyerB** [comment]
Thanks for you help, Jonathan!

I’m not a fan of  using ‘difference’ in the standard_name as we lose the fact of unnatural description when we take it instead of residuum.
We could add to my description: "‘balance’ means the sum of all contributions and sinks of soil moisture." as penultimate sentence for further clarification of the sign.

Have a nice day
Beate

**Comment by JonathanGregory** [comment]
Dear Beate @GeyerB 

Sorry, I'm not sure what you mean by "we lose the fact of unnatural description". Please could you explain? My concerns with "residuum" are

* It's not a familiar word. I don't think everyone would immediately know you mean the difference between the net value and the sum of the contributions.

* It doesn't indicate the sign convention. As you know, we try to make sign conventions explicit in standard names.

Best wishes

Jonathan

**Comment by taylor13** [comment]
I would agree that indication of the sign should be included in the name.  I think some variant on "difference_between_change_in_soil_moisture_content_and_sum_of_estimated_contributions" would work (if we all agree that difference between A and B means A-B and not B-A).   I might move "estimate" to rewrite it  "error_difference_between_estimated_changes_in_soil_moisture_content_and_sum_of_contributing_fluxes_and_sources".  Perhaps its too long, but is this what is meant?

[Somewhat tangentially, I'm surprised that in (well formulated) models the difference should be anything but zero.  I can understand that *observational* estimates of soil water content and the fluxes of water into the soil might be independently obtained, and so they wouldn't necessarily be consistent.  In a model don't the fluxes (i.e., the contributors to the tendency) determine the tendency (and the content) so conservation is assured?]
